meta/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
 
 dependencies:
+  - role: debops.apt_preferences
+    tags: [ 'depend::apt_preferences', 'type::dependency', 'depend-of::gitlab', 'type::dependency' ]
+    apt_preferences_dependent_list:
+      - package: 'golang-*'
```

Comment:
There is a `golang` package when you look in all packages, and not just installed ones:
```
# apt-cache search 'golang*'
golang - Go programming language compiler - metapackage
golang-dbg - Go programming language compiler - debug files
golang-doc - Go programming language compiler - documentation
golang-go - Go programming language compiler
golang-mode - Go programming language - mode for GNU Emacs
golang-src - Go programming language compiler - source files
dh-golang - debhelper add-on for packaging software written in Go (golang)
golang-go-darwin-386 - Go standard library compiled for darwin_386
golang-go-darwin-amd64 - Go standard library compiled for darwin_amd64
golang-go-freebsd-386 - Go standard library compiled for freebsd_386
golang-go-freebsd-amd64 - Go standard library compiled for freebsd_amd64
golang-go-freebsd-arm - Go standard library compiled for freebsd_arm
golang-go-linux-386 - Go standard library compiled for linux_386
golang-go-linux-amd64 - Go standard library compiled for linux_amd64
golang-go-linux-arm - Go standard library compiled for linux_arm
golang-go-netbsd-386 - Go standard library compiled for netbsd_386
golang-go-netbsd-amd64 - Go standard library compiled for netbsd_amd64
golang-go-windows-386 - Go standard library compiled for windows_386
golang-go-windows-amd64 - Go standard library compiled for windows_amd64
```